README.md

```diff
@@ -29,6 +30,7 @@ you are new to Vagrant, read on.
 - [Email Configuration](#email-configuration)
 - [Drush Alias Autoconfiguration](#drush-alias-autoconfiguration)
 - [Verifying basebox integrity](#verifying-basebox-integrity)
+  - [Verifying the download when adding the box](#verifying-the-download-when-adding-the-box)
```

Comment:
I'm guessing this is just something that was missed in a previous PR?

Reply:
Yes, indeed.